include/jemalloc/jemalloc_macros.h.in

```diff
@@ -142,7 +142,7 @@
 #  define JEMALLOC_COLD
 #endif
 
-#if (defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__)) && !defined(JEMALLOC_NO_RENAME)
+#if (defined(__APPLE__) || defined(__FreeBSD__) || defined(__OpenBSD__) || (defined(__linux__) && !defined(__GLIBC__))) && !defined(JEMALLOC_NO_RENAME)
```

Comment:
It looks this would also cover the android case? https://github.com/jemalloc/jemalloc/blob/dev/configure.ac#L710

@davidtgoldblatt I probably trust you more than myself here.